filters/all.js

```diff
@@ -202,28 +202,42 @@ module.exports = ({ Nunjucks }) => {
 
     //console.log('fixType: ' + name + ' ' + type + ' ' + JSON.stringify(property._json) + ' ' );
     //console.log("");
-
+  
     // If a schema has a property that is a ref to another schema,
     // the type is undefined, and the title gives the title of the referenced schema.
     let ret;
     if (type === undefined) {
       if (property._json.enum) {
         ret = _.upperFirst(javaName);
       } else {
-        ret = property.title();
+        // check to see if it's a ref to another schema.
+        ret = property._json['x-parser-schema-id'];
```

Comment:
You can use the public method `ext` or `extension`:

```suggestion
        ret = property.ext('x-parser-schema-id');
```